Harvest.Web/ClientApp/src/Tickets/TicketAttachments.tsx

```diff
@@ -62,25 +66,30 @@ export const TicketAttachments = (props: Props) => {
       </ul>
       <FormGroup>
         <Label>Attach files?</Label>
-        <FileUpload
-          files={ticketLoc.newAttachments || []}
-          setFiles={(f) => {
+        {ticket.completed !== false ? (
+          <FileUpload
+            files={ticketLoc.newAttachments || []}
+            setFiles={(f) => {
               setTicketLoc((ticket) => ({ ...ticket, newAttachments: [...f] }));
               updateFiles(f);
-          }}
-          updateFile={(f) =>
-            setTicketLoc((ticket) => {
-              // update just one specific file from ticket p
-              ticket.newAttachments[
-                ticket.newAttachments.findIndex(
-                  (file) => file.identifier === f.identifier
-                )
-              ] = { ...f };
+            }}
+            updateFile={(f) =>
+              setTicketLoc((ticket) => {
+                // update just one specific file from ticket p
+                ticket.newAttachments[
+                  ticket.newAttachments.findIndex(
+                    (file) => file.identifier === f.identifier
+                  )
+                ] = { ...f };
```

Comment:
this doesn't quite seem right to me... usually in a set hook you are supposed to return the new value, not just modify the object.

Reply:
Not sure how to change that. But it is working.